I'm sorry, but an uncaught exception occurred.

While running game code:
IOError: [Errno 2] No such file or directory: u'D:\\IE221.M21.CNCL\\Student Life/game\\saves\\persistent.1650944182.tmp'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/bootstrap.py", line 331, in bootstrap
    renpy.main.main()
  File "renpy/main.py", line 665, in main
    renpy.persistent.update(True)
  File "renpy/persistent.py", line 395, in update
    save()
  File "renpy/persistent.py", line 411, in save
    renpy.loadsave.location.save_persistent(data)
  File "renpy/savelocation.py", line 496, in save_persistent
    l.save_persistent(data)
  File "renpy/savelocation.py", line 347, in save_persistent
    with open(fn_tmp, "wb") as f:
IOError: [Errno 2] No such file or directory: u'D:\\IE221.M21.CNCL\\Student Life/game\\saves\\persistent.1650944182.tmp'

Windows-10-10.0.22000
Ren'Py 7.4.11.2266
Student Life 1.0
Tue Apr 26 10:47:20 2022
